config/dev/containers/provapp.yml

```diff
@@ -70,7 +70,7 @@ spec:
   - name: pbserver
     args:
     - --port=5002
-    - --etcd_endpoints=localhost:5003
+    - --db_path=file::memory:?cache=shared
```

Comment:
just curious, doe this mean for testing the sqlite file is entirely in memory?

Reply:
yep. I will add support for testing this using a file database in a future PR.